### PR TITLE
js-search: add definition

### PR DESCRIPTION
**Pull request by guoyunhe**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [ ] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [ ] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [ ] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not provide its own types, and you can not add them.
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [ ] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [ ] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, and `strictNullChecks` set to `true`.

If changing an existing definition:
- [ ] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [ ] Increase the version number in the header if appropriate.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dslint/dt.json" }`.

If removing a declaration:
- [ ] If a package was never on DefinitelyTyped, you don't need to do anything. (If you wrote a package and provided types, you don't need to register it with us.)
- [ ] Delete the package's directory.
- [ ] Add it to `notNeededPackages.json`.

### CODE REVIEW

File: types/js-search/tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": [
+            "es6"
+        ],
+        "noImplicitAny": true,
+        "noImplicitThis": true,
+        "strictNullChecks": false,
```
Comment (andy-ms) [inline types/js-search/tsconfig.json]: Make this `true`.

**Review by ghost — DISMISSED** [review]
Needs a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

**Comment by andy-ms** [comment]
Don't know why travis isn't showing up. See lint errors with `npm run lint js-search`.

**Comment by guoyunhe** [comment]
I got this error message but I do not understand what it means:

```
npm run lint js-search

> definitely-typed@0.0.1 lint /home/sign/git/DefinitelyTyped
> dtslint types "js-search"

Object.entries is not a function
```

**Comment by andy-ms** [comment]
Looks like you're running an older version of node.js.

**Pull request merged**